physionet-django/events/models.py

```diff
@@ -153,3 +155,33 @@ def withdraw(self, comment_to_applicant=None):
             comment_to_applicant (str): The comment to add to the participant about the status.
         """
         self._apply_decision(self.EventApplicationStatus.WITHDRAWN, comment_to_applicant)
+
+
+class EventAgreement(models.Model):
+    name = models.CharField(max_length=100)
+    slug = models.SlugField(max_length=120, unique=True)
+    version = models.CharField(max_length=15, default='', validators=[validate_version])
+    is_active = models.BooleanField(default=True)
+    html_content = SafeHTMLField(default='')
+    access_template = SafeHTMLField(default='')
+    creator = models.ForeignKey("user.User", on_delete=models.CASCADE)
+
+    class Meta:
+        default_permissions = ('add',)
+        unique_together = (('name', 'version', 'creator'),)
```

Comment:
I suggest changing this to `(('name', 'version'))`. This is more consistent with the DUA model, and we don't want to have to start worrying about "Alistair's" agreement vs. "Amit's" agreement.

Reply:
For sure.  quick question, wouldn't this lead to a case where Event Host "A" might endup editing an Event Agreement that is being used by user "B" on another Event.(but i suppose for someone with access to group "Event Host", we would expect them to be responsible.)

Reply:
Yes, that's true, but we likely wouldn't give event hosts permission to create agreements in the same way that project publishers cannot create data use agreements.

Reply:
Okay. i see. That makes sense. Admins will create the Event Agreement.